webapp/components/topbar/Topbar.js

```diff
@@ -176,6 +177,14 @@ class Topbar extends Component {
             />
           );
           break;
+        case TOPBAR_CONSTANTS.NETWORK_MODEL:
+          content = (
+            <ModelNetworkDialog
```

Comment:
When I click on `Instantiate` in the modal it and try to open the modal again it won't show up